Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -1277,6 +1281,7 @@ public FileformatInterface readMetadataFile(Process process)
             ff.read(serviceManager.getFileService().getFile(metadataFileUri).toString());
         } catch (ReadException e) {
             if (e.getMessage().startsWith("Parse error at line -1")) {
+                logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.